samples/core/component_build/component_build.ipynb

```diff
@@ -0,0 +1,368 @@
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2019-08-07 18:27:26:INFO:Build component complete.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from kfp import compiler\n",
+    "\n",
+    "add_op = compiler.build_python_component(\n",
+    "    component_func=add,\n",
+    "    staging_gcs_path=OUTPUT_DIR,\n",
+    "    dependency=[kfp.compiler.VersionedDependency(name='google-api-python-client', version='1.7.0')],\n",
+    "    base_image=BASE_IMAGE,\n",
+    "    target_image=TARGET_IMAGE)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Build a pipeline using this component"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import kfp.dsl as dsl\n",
+    "@dsl.pipeline(\n",
+    "   name='Calculation pipeline',\n",
+    "   description='A sample pipeline that performs arithmetic calculations.'\n",
+    ")\n",
+    "def calc_pipeline(\n",
+    "   a='1',\n",
+    "   b='7',\n",
+    "   c='17',\n",
+    "):\n",
+    "    #Passing pipeline parameter and a constant value as operation arguments\n",
+    "    add_task = add_op(a, b) #Returns a dsl.ContainerOp class instance. \n",
+    "    \n",
+    "    #You can create explicit dependancy between the tasks using xyz_task.after(abc_task)\n",
+    "    add_2_task = add_op(b, c)\n",
+    "    \n",
+    "    add_3_task = add_op(add_task.output, add_2_task.output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Complie the pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline_func = calc_pipeline\n",
+    "pipeline_filename = pipeline_func.__name__ + '.pipeline.zip'\n",
+    "\n",
+    "import kfp.compiler as compiler\n",
+    "compiler.Compiler().compile(pipeline_func, pipeline_filename)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Submit the pipeline for execution"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "Run link <a href=\"/pipeline/#/runs/details/01df9e80-b941-11e9-b782-42010a8000a3\" target=\"_blank\" >here</a>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#Specify pipeline argument values\n",
+    "arguments = {'a': '7', 'b': '8'}\n",
+    "\n",
+    "#Submit a pipeline run\n",
+    "run_name = pipeline_func.__name__ + ' run'\n",
+    "run_result = client.run_pipeline(experiment.id, run_name, pipeline_filename, arguments)\n",
+    "\n",
+    "#This link leads to the run information page. \n",
+    "#Note: There is a bug in JupyterLab that modifies the URL and makes the link stop working"
```

Comment:
From what I know it's back to work now. Did you observe any issue?

Reply:
It works! thank you :) removing the comment.